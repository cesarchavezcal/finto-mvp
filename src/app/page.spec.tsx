import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Home from './page';

describe('Home', () => {
  beforeEach(async () => {
    // Action
    const Component = await Home();
    render(Component);
  });

  it('renders the heading', async () => {
    // Arrange
    const el = screen.queryByRole('heading');
    // Assert
    expect(el).toBeInTheDocument();
    expect(el).toMatchSnapshot();
  });

  it('heading should show "Hello World!"', async () => {
    // Arrange
    const el = screen.queryByRole('heading');
    // Assert
    expect(el).toHaveTextContent('Hello World');
    expect(el).toMatchSnapshot();
  });
});
