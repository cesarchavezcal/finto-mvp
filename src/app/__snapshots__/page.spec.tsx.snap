// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home heading should show "Hello World!" 1`] = `
<h1
  role="heading"
>
  Hello World
</h1>
`;

exports[`Home renders the heading 1`] = `
<h1
  role="heading"
>
  Hello World
</h1>
`;
